src/geom/SkyWcs.cc

```diff
@@ -334,7 +334,26 @@ std::shared_ptr<typehandling::Storable> SkyWcs::cloneStorable() const {
     return std::make_unique<SkyWcs>(*this);
 }
 
-std::string SkyWcs::toString() const { return "SkyWcs"; }
+std::string SkyWcs::toString() const {
+    std::ostringstream os;
+    if (isFits()) {
+        os << "FITS standard SkyWcs:";
+    } else {
+        os << "Non-standard SkyWcs (Frames: ";
+        // Print the frames in index order (frames are numbered from 1).
+        std::string delimiter = "";
+        for (size_t i = 1; i <= getFrameDict()->getAllDomains().size(); ++i) {
+            os << delimiter << getFrameDict()->getFrame(i)->getDomain();
+            delimiter = ", ";
+        }
```

Comment:
This looks very complicated. Could you add a description or an example to the documentation comments for `toString` and `operator<<`? At the very least, it would be helpful for the user to know that the string has line breaks (so, e.g., you shouldn't print a `SkyWcs` in a sentence).

Handy convention from the language that shall not be named: "The exact details of the representation are unspecified and subject to change, but the following may be regarded as typical:"

Reply:
I put in the newlines because it seemed much more readable that way. For complicated objects, I'm never quite sure whether to try to keep it on one line, or expand it into a multi-line but more readable thing.

Reply:
I've added an example of what the output could look like to the `toString()` docs.